ansible/group_vars/all/vars.yml

```diff
@@ -47,3 +48,4 @@ passwd:
 ansible_venv: /root/venv
 docker_apt_ver: ""
 airflow_tag: "1.8.0-oo3"
+oomsm_tag: "1.0.0-beta.1"
```

Comment:
That's good idea to move it to more reasonable location, I just reused `airflow_tag` location.
- [ ] move tag